### PR TITLE
Documentation update

### PR DESCRIPTION
**Pull request by bjrmatos**
i hope you like it :)

### CODE REVIEW

**Comment by yaronn** [comment]
Thanks!  I will check PRs and issues later this month when I have more time
and will let you know if I'll have questions.

**Comment by bjrmatos** [comment]
Cool, let me know if i can help you with anything

**Comment by bjrmatos** [comment]
hi @brianhartsock would be awesome if you could review this PR.

The only thing left to do is document that `loadSignature` accepts a xml string or node (introduced in f1a95ffd24836)

English is not my native language, so would be great that you review if everything is fine :smile:

File: README.md [hunk 1]
```diff
@@ -10,7 +10,45 @@ Install with [npm](http://github.com/isaacs/npm):
...
+
+by default the following algorithms are used:
+
+*Canonicalization/Transformation Algorithm:* Exclusive Canonicalization http://www.w3.org/2001/10/xml-exc-c14n#
+
+*Hashing Algorithm:* SHA1 digest http://www.w3.org/2000/09/xmldsig#sha1
+
+*Signature Algorithm:* RSA-SHA1 http://www.w3.org/2000/09/xmldsig#rsa-sha1
+
+[You are able to extend xml-crypto with further algorithms.](#customizing-algorithms)
```
Comment (brianhartsock) [inline README.md]: Should this say "You are able to extend xml-crypto with custom algorithms" or maybe "additional algorithms"

**Comment by brianhartsock** [comment]
:+1:

**Comment by bjrmatos** [comment]
updated :)

**Comment by brianhartsock** [comment]
:+1:

**Comment by bjrmatos** [comment]
ok docs updated.. i merge this, please let me know if something is wrong or can be improved! :smiley:

**Pull request merged**